core/src/main/java/org/prism/ParsingOptions.java

```diff
@@ -0,0 +1,82 @@
+package org.prism;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+// @formatter:off
+public abstract class ParsingOptions {
```

Comment:
this file comes from Prism main repo:
https://github.com/ruby/prism/blob/main/java/org/prism/ParsingOptions.java